src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/__tests__/MaskStateCellModelTests.swift

```diff
@@ -0,0 +1,109 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+import HealthCertificateToolkit
+import class CertLogic.Description
+
+class MaskStateCellModelTests: XCTestCase {
+
+    func testMaskStateFromWalletInfo_MaskOptional() throws {
+		// GIVEN
+		
+		let healthCertifiedPerson = HealthCertifiedPerson(healthCertificates: [])
+		
+		let titleMock = "Maskenbefreiung"
+		let subtitleMock = "Sie sind nicht von der Maskenpflicht ausgenommen"
+		let longTextMock = "Von der Maskenpflicht sind alle Personen befreit, die innerhalb der letzten 3 Monate geimpft wurden oder genesen sind oder innerhalb der letzten 24 Stunden negativ getestet wurden."
+		let faqAnchorMock = "maskstate"
+		
+		healthCertifiedPerson.dccWalletInfo = .fake(
+			maskState: .fake(
+				visible: true,
+				badgeText: .fake(string: ""),
```

Comment:
I think it's better to set dummy string too and check later instead of empty string.